api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -109,8 +130,14 @@ message CommonTlsContext {
         [(validate.rules).message = {required: true}];
 
     // Config for fetching validation context via SDS API.
-    SdsSecretConfig validation_context_sds_secret_config = 2
-        [(validate.rules).message = {required: true}];
+    SdsSecretConfig validation_context_sds_secret_config = 2 [
+      (validate.rules).message = {required: true},
+      (udpa.annotations.field_migrate).oneof_promotion = "dynamic_validation_context"
+    ];
+
+    // Config for fetching validation context via CA Manager plugin
```

Comment:
What is a CA manager? This concept hasn't been introduced yet.

Reply:
Meant to say CA Client plugin. Forgot to change this comment